java/ingest/src/main/java/sleeper/ingest/job/IngestJob.java

```diff
@@ -26,7 +27,11 @@ public class IngestJob {
 
     private IngestJob(Builder builder) {
         tableName = builder.tableName;
-        id = builder.id;
+        if (builder.id != null) {
+            id = builder.id;
+        } else {
+            id = UUID.randomUUID().toString();
```

Comment:
Can we log the random id that we have created? i.e. "Null job id provided, set id to abcd123456789"